lib/util/crop_compress_image.dart

```diff
@@ -6,7 +6,7 @@ import 'package:image_cropper/image_cropper.dart';
 
 class AppHelper {
```

Comment:
I see you changed the file name, change also name of this class.

Reply:
Nie zmieniłem nazwy pliku xD